src/library/auth/Authenticator.jsx

```diff
@@ -41,13 +39,13 @@ const CustomAuthenticator = ({ children, ...props }) => (
 );
 
 const ContentValidator = ({ children, authState }) => {
-  return <div >{authState === 'signedIn' && <div>{children}</div>}</div>;
+  return <div>{authState === 'signedIn' && <div>{children}</div>}</div>;
```

Comment:
This is going to display `<div>false</div>` if `authState === 'signedIn'` is `false`

Reply:
Also, are the outher `div`s needed?

```suggestion
  return authState === 'signedIn'
       ? <div>{children}</div>
       : null;
```

Reply:
Good question! I didn't check this logic, tbh. These lines were changed by prettier which re-formatted them.